packages/dd-trace/src/appsec/passport.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const log = require('../log')
+const { trackEvent } = require('./sdk/track_event')
+const { setUserTags } = require('./sdk/set_user')
+
+const UUID_PATTERN = '^[0-9A-F]{8}-[0-9A-F]{4}-[1-5][0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$'
+const regexUsername = new RegExp(UUID_PATTERN, 'i')
+
+const SDK_USER_EVENT_PATTERN = '_dd\\.appsec\\.events\\.users\\.[\\W\\w+]+\\.sdk'
```

Comment:
why no start and end anchors ?
```suggestion
const SDK_USER_EVENT_PATTERN = '^_dd\\.appsec\\.events\\.users\\.[\\W\\w+]+\\.sdk$'
```